cea/demand/electrical_loads.py

```diff
@@ -45,8 +45,7 @@ def calc_E_totals(Aef, Ealf, Eauxf, Edataf, Eprof, Eaf, Elf):
         Epro_tot = Edata_tot = 0
         Ealf = np.zeros(8760)
         Eprof = np.zeros(8760)
-        Edataf = np.zeros(8760)
-    return Ealf, Ealf_0, Ealf_tot, Eauxf_tot, Edataf, Edata_tot, Eprof, Epro_tot, Eaf_0, Elf_0, Eaf_tot, Elf_tot
```

Comment:
thanks for asking. I kind of made a little wrongdoing by fixing three issues in the same branch. These elements are not anymore used so they were erased. Th eelctrical calculation was simplified and improved.